api/helpers/uploadedFileErrorChecks.js

```diff
@@ -37,48 +44,97 @@ function initialiseClamScan(req) {
     } = req._sails.config.upload;
     const clamAvOptions = {
         debugMode: JSON.parse(clamavDebugEnabled) || false,
-        clamscan: {active: false},
+        clamscan: { active: false },
         clamdscan: {
             active: false,
             host: clamavHost,
             port: clamavPort,
         },
     };
-
-    return new NodeClam().init(clamAvOptions);
+    try {
+        return new NodeClam().init(clamAvOptions);
+    } catch (err) {
+        throw new Error(err);
+    }
 }
 
-async function virusScanAndCheckFiletype(req) {
+async function checkFileType(req) {
     try {
-        if (req.files.length === 0) {
-            req.session.eApp.uploadMessages.noFileUploadedError = true;
-            throw new Error('No files were uploaded.');
+        sails.log.info('Checking file type...');
+        for (const file of req.files) {
+            inDevEnvironment
+                ? await checkLocalFileType(file, req)
+                : await checkS3FileType(file, req);
+        }
+    } catch (err) {
+        if (err.message.includes('not a PDF')) {
+            throw new UserAdressableError(`checkFileType ${err}`);
         }
+        throw new Error(`checkFileType ${err}`);
```

Comment:
check slack - it would be better to use error codes rather than doing a string search